tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/LanguageExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test
+{
+    public static class LanguageExtensions
+    {
+        public static string GetFileExtension(this Language lang)
+        {
+            return lang switch
+            {
+                Language.CSharp => "cs",
+                Language.VisualBasic => "vb",
+                Language.FSharp => "fs",
```

Comment:
interesting.  do you support f#?

Reply:
nope, we have that in the enumeration so I figured I'd add it here to the extension method.

starting to think we should remove fsharp references so that the expectation is clear.